axelrod/strategies/worse_and_worse.py

```diff
@@ -10,11 +8,12 @@ class WorseAndWorse (Player):
     it is more and more likely to defect as the round goes on.
 
     Names:
-        - worse_and_worse: [PRISON1998]
+        - worse_and_worse: Source code available at the download tab of
+                            [PRISON1998].
```

Comment:
Does this indent render ok? (you can run `make html` to check from the `docs` directory, you might have to pip install a sphinx theme...)

I think the rst syntax should be:

```
        - worse_and_worse: Source code available at the download tab of
          [PRISON1998].
```